package.json

```diff
@@ -51,7 +51,7 @@
     "stylelint": "stylelint src/components/*/*.js",
     "stylelint:ci": "stylelint src/components/*/*.js --formatter github",
     "stylelint:fix": "stylelint src/components/*/*.js --fix",
-    "typecheck": "tsc -p tsconfig-typecheck-jsdoc.json"
+    "typecheck": "tsc -p tsconfig.ci.json"
```

Comment:
If I had to make a choice, I'd stick with the prefix if it's standard. And for the `ci`, well either it will be used within the CLI or in the CI. In either way you'll probably not check the config file when using the command. Thus said, I guess we can leave it with the `ci` prefix. Moreover, you can easily find the file from the package.json script if needed.